uptime-kuma.json

```
@@ -56,72 +219,91 @@
             {
               "options": {
                 "0": {
-                  "color": "red",
+                  "color": "dark-red",
                   "index": 0,
-                  "text": "DOWN"
+                  "text": "Down"
                 },
                 "1": {
-                  "color": "green",
+                  "color": "dark-green",
+                  "index": 2,
```

is the index `2` for id `"1"` intentional?

---

This should have no impact. Could be a leftover when creating the different color styles and later changed the order.
```suggestion
                  "index": 1,
```